scripts/tests/test_suites.sh

```diff
@@ -107,16 +110,14 @@ for j in "${iter_array[@]}"; do
         # tee expeditiously; otherwise it will buffer things up and we
         # will never see the string we want.
 
-        # Clear out our temp files so we don't accidentally do a stale
-        # read from them before we write to them.
-        rm -rf /tmp/"$application"-log
-        touch /tmp/"$application"-log
+        application_log_file=/tmp/test_suites_app_logs/"$application-$i-$j"-log
+        touch "$application_log_file"
```

Comment:
Please leave the `rm`, just in case.  There _should_ be nothing in `/tmp`, but better not to trust that.

Reply:
At the start of the script it's deleting the folder `/tmp/test_suites_app_logs`. That should cleanup the logs from any other runs. And, logs from the current run should not overlap any freshly created log file. Does that seem to address the issue?

Reply:
Yes, indeed.  I had forgotten we were deleting the whole folder!